scripts/components/OpenSearch-Dashboards/install.sh

```diff
@@ -72,20 +72,16 @@ fi
 [ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
 [ -z "$DISTRIBUTION" ] && DISTRIBUTION="tar"
 
+# Make sure the cwd is where the script is located
+DIR="$(dirname "$0")"
+echo $DIR
+cd $DIR
```

Comment:
So, why is this necessary?

Reply:
If this is not there we cannot find the script folder location.
If you call install script outside of its located folder, it will default cwd to other folder.

Reply:
Changing directories is generally not a good pattern, because it relies on that location. You could use `$DIR/script.sh` to call it for example. The script should probably be agnostic of where it's called from. 

It's not a huge deal, but generally having to `cd` hides bugs in my experience.

Reply:
This is due to the fact we are using relative paths right now for the copy.
A lot of the scripts we have right now is relying on cd.
Will definitely worth revisit this later.

Thanks.

Reply:
Where? Which scripts? I cannot find any instances of such.

Even if that's the case, can we please call these scripts from the right directory as a workaround instead of "fixing" it this way? I am worried this propagates into dozens of scripts and becomes the default pattern that we can never undo. I am 100% sure no other build.sh/install.sh does this and we're going to have to change all of them.

Reply:
@dblock I did not add these lines, these are existing lines and we have been using this method since ODFE.

Examples:
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/opensearch-onetime-setup.sh#L12
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/deploy_single_tar_cluster.sh#L18
https://github.com/opensearch-project/opensearch-build/blob/main/scripts/components/OpenSearch-Dashboards/install.sh#L72-L74